packages/react/src/LogoSuite/LogoSuite.tsx

```diff
@@ -162,11 +162,14 @@ type LogoSuiteLogoBarProps = BaseProps<HTMLDivElement> & {
 
 const _LogoBar = forwardRef(
   (
-    {className, children, marquee = false, marqueeSpeed = 'normal', variant = 'muted', ...props}: LogoSuiteLogoBarProps,
+    {className, children, marquee = false, marqueeSpeed = 'normal', variant, ...props}: LogoSuiteLogoBarProps,
     ref: Ref<HTMLDivElement>,
   ) => {
     const [prefersReducedMotion, setPrefersReducedMotion] = useState(false)
 
+    const childrenCount = React.Children.toArray(children).length
+    variant ??= childrenCount <= 5 ? 'emphasis' : 'muted'
+
```

Comment:
As this is the main logic being added, could we add a test case for this please? Ideally in this PR, or in a follow up would be great 🙏.

Reply:
Added, great suggestion thanks!